pkg/driverbuilder/builder/amazonlinux.go

```diff
@@ -179,6 +180,14 @@ func (a *amazonlinux2) URLs(kr kernelrelease.KernelRelease) ([]string, error) {
 	return fetchAmazonLinuxPackagesURLs(a, kr)
 }
 
+func (a *amazonlinux2) GCCVersion(kr kernelrelease.KernelRelease) semver.Version {
```

Comment:
Implement `GCCVersionRequestor` interface to override gcc used for 5.10 kernels (enforcing gcc-10).